dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,193 @@
+                            "title": "Data Quality",
+                            "text": "Inaccurate, inconsistent, or outdated data jeopardizes the integrity of analytics and decision processes, leading to misguided strategies."
+                        },
+                        {
+                            "icon": "/static/icons/3d-mpr-toggle.svg",
+                            "title": "Data Usage Rights",
+                            "text": "Companies must ensure they have the legal permissions to collect, store, process, and share data while adhering to regulations and privacy laws. "
+                        },
+                        {
+                            "icon": "/static/icons/Airline--rapid-board.svg",
+                            "title": "Data Usage",
+                            "text": "Many organizations struggle with underutilization. Data analysis tools might be intimidating or inaccessible to certain teams, limiting the ability to extract valuable insights. "
+                        }
+                    ]
+                },
+                {
+                    "title": "How it works",
+                    "text": [
+                        "How does Dataland work?"
+                    ],
+                    "cards": [
+                        {
+                            "title": "Search",
+                            "text": "Utilize the search feature to find specific companies within the platform."
+                        },
+                        {
+                            "title": "Request company’s inclusion",
+                            "text": "Request company’s inclusion in case you can’t find the company you are looking for."
+                        },
+                        {
+                            "title": "Request framework data",
+                            "text": "Create or join a campaign to get the missing data."
+                        },
+                        {
+                            "title": "Download",
+                            "text": "Download either through the platform or through API."
+                        }
+                    ]
+                },
+                {
+                    "title": "Claim",
+                    "text": [
+                        "In Dataland, ",
+                        "you",
+                        " have the power."
+                    ]
+                },
+                {
+                    "title": "Join a campaign",
+                    "text": [
+                        "Join a campaign to",
+                        " to get the missing ",
+                        "data.",
+                        "The Proof of Concepts are a simple way for you to get better data coverage and improved data quality, while testing Dataland."
+                    ],
+                    "cards": [
+                        {
+                            "title": "Pathway to Paris",
```

Comment:
I think it is Pathway**s** to Paris

Reply:
It looks like we have got this wrong on multiple occurences. All, I think. However, the internet does not have a consense on this.